src/fetchUpdates.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+
+(async () => {
+  try {
+    const headers = {
+      headers: { Accept: 'application/vnd.github.v3.raw' },
+    };
+
+    const response = await fetch(
+      `https://api.github.com/repos/ApollosProject/apollos-prototype/contents/version.json`,
+      headers,
+    );
+
+    const { mostRecentVersion, newestVersion } = await response.json();
```

Comment:
I like the idea of this, however it doesn't work :/ We don't have a `version.json` on the root of our project. You could look at `lerna.json` instead perhaps

Reply:
Correct. This would be an added file we would need. The issue with lerna.json is the lack of both version variables. A versions file could be used here and could even be used for multiple other purposes

Reply:
What's the difference between mostRecentVersion and newestVersion? I think I would rather use a `currentVersion` and omit the other value.

Reply:
Also, I totally missed your reference to it being not-yet created in the issue description :D